src/components/Selectors/selectors.module.scss

```diff
@@ -0,0 +1,262 @@
+.selector-wrapper {
```

Comment:
Please adjust styles according to the [Figma](https://www.figma.com/file/U4t894XWtt2xgDBA2rIeK9/Octuple-DS?node-id=702%3A2282)